packages/alfa-puppeteer/test/puppeteer.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { Query } from "@siteimprove/alfa-dom";
+import { test } from "@siteimprove/alfa-test";
+import * as path from "node:path";
+import * as url from "node:url";
+
+import puppeteer from "puppeteer";
+import { Puppeteer } from "../dist/puppeteer.js";
+
+// TODO: This should be replaced with import.meta.dirname once we switch to Node 22
+const __filename = url.fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const fixture = path.join(__dirname, "fixture");
+
+test(".toPage() scrapes a page", async (t) => {
+  const browser = await puppeteer.launch({
+    headless: true,
+    args: ["--no-sandbox", "--disable-dev-shm-usage"],
+  });
+
+  const page = await browser.newPage();
+
+  // Navigate to the page to scrape
+  await page.goto(url.pathToFileURL(path.join(fixture, "page.html")).href);
+
+  // Retrieve the viewport dimensions
+  const { width, height } = await page.evaluate(() => ({
+    width: window.document.documentElement.clientWidth,
+    height: window.document.documentElement.clientHeight,
+  }));
```

Comment:
**Question:** Are these stable enough under puppeteer to be able to hardcode them in tests?